biogeochem/EDPatchDynamicsMod.F90

```diff
@@ -2603,15 +2622,19 @@ subroutine terminate_patches(currentSite)
 
     currentPatch => currentSite%youngest_patch
     do while(associated(currentPatch)) 
-       
+       oldercpatch => currentPatch%older       
        if(currentPatch%area <= min_patch_area)then
-          
+
+         if(hlm_use_nocomp.eq.ifalse)then !just fuse to older or younger patch
```

Comment:
see below -- I am not sure we want to use alternate logic here, but instead treat both the anthro_disturbance_label and the nocomp_pft_label as similarly as possible